python/graphstorm/eval/evaluator.py

```diff
@@ -749,20 +789,22 @@ def __init__(self, eval_frequency,
             early_stop_strategy)
 
     def compute_score(self, pred, labels, train=True):
-        """ Compute evaluation score
-
-            Parameters
-            ----------
-            pred:
-                Rediction result
-            labels:
-                Label
-            train: boolean
-                If in model training.
-
-            Returns
-            -------
-            Evaluation metric values: dict
+        """ Compute feature reconstruction evaluation scores.
+
+        Parameters
+        ----------
+        pred: 2D tensor
+            The 2D tensor stores the prediction results.
+        labels: 2D tensor
+            The 2D tensor stores the labels, which stores the original features.
```

Comment:
"The 2D tensor stores the labels, which stores the original features." looks confusing.

Reply:
Updated to `The 2D tensor stores the labels that are the original node features as this is a feature reconstruction task.`